src/commands/OSRS_Fun/dryCalc.ts

```diff
@@ -0,0 +1,45 @@
+import { Command, CommandStore, KlasaMessage } from 'klasa';
+
+/* 
+  Witty Switch Comments Pulled From:
+  https://oldschool.runescape.wiki/w/Calculator:Dry_calc
+*/
+export default class extends Command {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 1,
+			description:
+				'Calculates the drop chance of getting an item within a set amount of rolls.',
+			usage: '<dropRate:int> <rolls:int>',
+			usageDelim: ' '
+		});
+	}
+
+	async run(msg: KlasaMessage, [dropRate = 1, rolls = 1]: [number, number]) {
+		if (dropRate > 100000000 || rolls > 100000000) {
+			throw "I can't process a number higher than 100 million! enneUni has already claimed the only uncut onyx!";
+		}
+		if (dropRate < 1 || rolls < 1) {
+			throw "I can't calculate with zeros nor negative numbers. Do more rolls or update the drop values.";
+		}
```

Comment:
Remove both of these checks, and instead edit the `usage` option to this: `usage: '<dropRate:int{1,1000000}> <rolls:int{1,1000000}>',` which will handle those checks for you, after doing this also remove the `=1` default assignments